### PR TITLE
[RF] POM | AT_17.01.001 | Org folder > Configure > Change status folder to disable

### PR DESCRIPTION
**Pull request by asyniagina**
https://trello.com/c/EGnWJFhZ/1587-rf-pom-at1701001-organization-folder-configure-organization-folder-change-status-folder-to-disable

### CODE REVIEW

**Comment by asyniagina** [comment]
[[RF] POM | AT_17.01.001 | Organization folder > Configure organization folder > Change status folder to disable](https://trello.com/c/EGnWJFhZ/1587-rf-pom-at1701001-organization-folder-configure-organization-folder-change-status-folder-to-disable)

**Review by SawkaQA — APPROVED** [review]

File: cypress/e2e/orgFolderConfigurate.cy.js
Comment (LanaBondarenko) [inline cypress/e2e/orgFolderConfigurate.cy.js:1]: As mentioned in the lecture, we should not delete previous tests, but only skip them.

File: cypress/pageObjects/HomePage.js
Comment (LanaBondarenko) [inline cypress/pageObjects/HomePage.js:1]: It seems to me that the createNewOrgFolder(name) does not belong to the homepage. Probably would be better to move it to NewItemPage.

File: cypress/pageObjects/OrgFolderPage.js [hunk 1]
```diff
@@ -1,12 +1,20 @@
 import HomePage from "./HomePage";
+import OrgFolderConfigurePage from "./OrgFolderConfigurePage";
 
 class OrgFolderPage {
     getDashboard = () => cy.get('#breadcrumbs a').contains("Dashboard");
+    getConfigureTheProjectLink = () => cy.get('.content-block [href="./configure"]');
+    getDisabledProject = () => cy.get('#enable-project');
```
Comment (Rom-T) [inline cypress/pageObjects/OrgFolderPage.js:7]: Please rename name to `getEnableProjectForm`

File: cypress/pageObjects/HomePage.js [hunk 2]
```diff
@@ -36,5 +38,23 @@ class HomePage {
...
+    homePage
+            .clickNewItemSideMenuLink()
+            .typeNewItemNameInputField(name)
+            .selectOrgFolderItem()
+            .clickOkBtnAndGoOrgFolderConfig()
+            .clickSaveBtnAndGoOrgFolder()
+            .clickGoToDashboard()
+            .getMainPanel()
+            .should('contain.text', name);
+  }
```
[the remark this answers is not in the document]
Comment (asyniagina) [reply cypress/pageObjects/HomePage.js:58]: > Collaborator

But I use this code(create orgFolder) a lot of time in my next tests. It is a precondition. And I asked about it on last Stan lection (2.14:00 – вопрос о промежуточных assert при переходе на РОМ. Перенос их в метод в command
). Maybe I don't understand correctly. Please, help me, with where I should move this method. Thank you

**Review by Rom-T — APPROVED** [review]

**Review by LanaBondarenko — APPROVED** [review]

File: cypress/e2e/orgFolderConfigurate.cy.js
Comment (asyniagina) [reply cypress/e2e/orgFolderConfigurate.cy.js:1]: I fixed. Thank you for your review

File: cypress/pageObjects/HomePage.js
Comment (asyniagina) [reply cypress/pageObjects/HomePage.js:1]: @LanaBondarenko 
I fixed. Thank you for your review

File: cypress/pageObjects/OrgFolderPage.js (hunk 1 above)
Comment (asyniagina) [reply cypress/pageObjects/OrgFolderPage.js]: @Rom-T 
I fixed all according to the comments. Thank you very much for your review

**Pull request merged**